src/libs/actions/Transaction.ts

```diff
@@ -58,6 +59,7 @@ function addStop(transactionID: string) {
 }
 
 function saveWaypoint(transactionID: string, index: string, waypoint: RecentWaypoint | null, isDraft = false) {
+    IOU.setMoneyRequestAmount_temporaryForRefactor(transactionID, 0, '');
```

Comment:
I would suggest putting this in a new function, `resetMoneyRequestAmount`. It would give us a place to comment on the `0` constant, explaining that it's a special value.

Reply:
@cubuspl42 Updated, please help to check again